client/src/styles/PropertiesSummary.scss

```diff
@@ -73,41 +73,42 @@
         }
       }
 
-      .warning-banner {
-        background-color: $wowza-orange;
-        color: $wowza-dark;
-        border-radius: 0.2rem;
+      .MapBanner {
+        display: flex;
+        position: relative;
+        min-height: 5.4rem;
         padding: 0.911rem 2.1rem 1.012rem 1.08rem;
         margin-bottom: 1.5rem;
-        display: flex;
+        border-radius: 0.2rem;
+        font-family: "Inconsolata", monospace;
         font-size: 1.4rem;
-        max-width: 47.5rem;
 
         @include for-tablet-portrait-up {
           position: absolute;
           z-index: 100;
           bottom: 1rem;
           left: 1rem;
           margin-bottom: 0;
+          width: 37.5rem;
         }
 
-        .float-left {
-          display: flex;
-          align-items: center;
-          padding: 0;
-        }
-
-        .icon {
-          width: 2.536rem;
-          height: 2.277rem;
-          margin-right: 1.012rem;
+        .MapBanner__message {
```

Comment:
it looks like a chunk of the PropertiesSummary css is duplicated from PropertiesMap, wondering if that can be factored into a shared file or something? might be useful if it's common to the MapBanner and we end up reusing the MapBanner component elsewhere (def could also be done later)

Reply:
oh yeah for sure, good idea - just did that.